docs/quick-start.md

```diff
@@ -80,278 +82,274 @@ Save the above `Provider` as `provider.yaml`, save your Google Application
-publicIP:                             13 bytes
-serverCACertificateCertSerialNumber:  1 bytes
-serverCACertificateCreateTime:        24 bytes
-serverCACertificateExpirationTime:    24 bytes
-serverCACertificateSha1Fingerprint:   40 bytes
-endpoint:                             13 bytes
-serverCACertificateCert:              1272 bytes
+endpoint:  12 bytes
+port:      4 bytes
 ```
 
 That's all there is to static provisioning with Crossplane! We've created a
-`CloudSQLInstance` as cluster scoped infrastructure, then claimed it as a
-`MySQLInstance`. You can use `kubectl describe` to view the detailed
-configuration and status of your `CloudSqlInstance`.
+`CloudMemorystoreInstance` as cluster scoped infrastructure, then claimed it as
+a `RedisCluster`. You can use `kubectl describe` to view the detailed
+configuration and status of your `CloudMemorystoreInstance`.
 
 ```bash
-$ kubectl describe example-cloudsql-instance
-Name:         example-cloudsql-instance
-Annotations:  crossplane.io/external-name: example-cloudsql-instance
-API Version:  database.gcp.crossplane.io/v1beta1
-Kind:         CloudSQLInstance
+$ kubectl describe cloudmemorystoreinstance example-cloudmemorystore-instance
+Name:         example-cloudmemorystore-instance
+Namespace:    
+Labels:       <none>
+Annotations:  crossplane.io/external-name: example-cloudmemorystore-instance
+              kubectl.kubernetes.io/last-applied-configuration:
+                {"apiVersion":"cache.gcp.crossplane.io/v1beta1","kind":"CloudMemorystoreInstance","metadata":{"annotations":{},"name":"example-cloudmemory...
+API Version:  cache.gcp.crossplane.io/v1beta1
+Kind:         CloudMemorystoreInstance
+Metadata:
+  Creation Timestamp:  2019-11-20T17:16:27Z
+  Finalizers:
+    finalizer.managedresource.crossplane.io
+  Generation:        4
+  Resource Version:  284706
+  Self Link:         /apis/cache.gcp.crossplane.io/v1beta1/cloudmemorystoreinstances/example-cloudmemorystore-instance
+  UID:               7c9cb407-0bb9-11ea-8195-42010a800088
 Spec:
+  Claim Ref:
+    API Version:  cache.crossplane.io/v1alpha1
+    Kind:         RedisCluster
+    Name:         example-redis-claim
+    Namespace:    default
+    UID:          9cd9105b-0bb9-11ea-8195-42010a800088
   For Provider:
-    Database Version:  MYSQL_5_6
-    Gce Zone:          us-west2-b
-    Instance Type:     CLOUD_SQL_INSTANCE
-    Region:            us-west2
-    Settings:
-      Activation Policy:  ALWAYS
-      Backup Configuration:
-        Start Time:       17:00
-      Data Disk Size Gb:  10
-      Data Disk Type:     PD_SSD
-      Ip Configuration:
-        ipv4Enabled:  true
-      Location Preference:
-        Zone:               us-west2-b
-      Pricing Plan:         PER_USE
-      Replication Type:     SYNCHRONOUS
-      Storage Auto Resize:  true
-      Tier:                 db-n1-standard-1
+    Alternative Location Id:  us-west2-b
+    Authorized Network:       projects/crossplane-playground/global/networks/default
```

Comment:
We should avoid leaking our playground project's name in our docs and examples. I suggest finding and replacing `crossplane-playground` with some other value.